src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -56,6 +57,24 @@ export const availableActions: AvailableActionsType = {
     },
 };
 
+export const volunteerAvailableActions: Omit<AvailableActionsType, "Delete Parcel"> = {
```

Comment:
Why is this exported? Where else is it used in?

Reply:
Removed :)